src/world/GameMop/Management/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ set(PATH_PREFIX GameMop/Management)
 set(SRC_GAMEMOP_MANAGEMENT_FILES
     ${PATH_PREFIX}/GuildFinderMgr.cpp
     ${PATH_PREFIX}/GuildFinderMgr.h
-    ${PATH_PREFIX}/QuestMgr.Cata.cpp
+    # ${PATH_PREFIX}/QuestMgr.Cata.cpp
```

Comment:
If this file is no longer used, delete this line.

Reply:
delete file (QuestMgr.Cata.cpp)